MapboxCoreNavigation/MBXPeerWrapper.h

```diff
@@ -0,0 +1,5 @@
+#import <Foundation/Foundation.h>
+
+@interface MBXPeerWrapper
+
+@end
```

Comment:
We might not even need to declare the interface; it might be enough to simply forward-declare the class:

```suggestion
@class MBXPeerWrapper;
```

Reply:
Actually, looks like that doesn’t work. The forward declaration doesn’t satisfy the Swift compiler, even if it appears in the umbrella header.